### PR TITLE
Add D8VK capabilities to Bottles

### LINKED ISSUE #2641: [Request]: Add D8VK support
### Tell us the problem or your need

Currently, there is no way to run Direct3D 8 titles on Vulkan on Bottles.

### Describe the solution you'd like

I'd like to propose integrating D8VK. I don't really know much about it, but we could use it and make it available for those who want to play Direct3D 8 on Vulkan.

### Other solutions?

_No response_

### Additional context and references

Project: https://github.com/AlpyneDreams/d8vk

### PR DESCRIPTION
**Pull request by Kinsteen**
# Description
Title is pretty self explanatory.

To test right now, you need to have local components with D8VK, or you can have locally installed it like so:
image

Fixes #2641 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. 
Provide instructions so we can reproduce.
- [x] locally with a random Direct3D 8 game I found [here](http://daidai.moo.jp/accelerationofsuguri.html)

### CODE REVIEW

**Comment by Kinsteen** [comment]
@AlpyneDreams, here is D8VK added :) If you'd like to test with more games, I'll be very happy if you can assure it works correctly!

**Comment by TheEvilSkeleton** [comment]
I blocked the MR, because, as discussed a few months ago, we're focusing on fixing bugs and polishing.

File: bottles/frontend/windows/launchoptions.py [hunk 1]
```diff
@@ -174,12 +191,13 @@ def get_config(self):
...
     def __idle_save(self, *_args):
         dxvk = self.switch_dxvk.get_state()
+        d8vk = self.switch_d8vk.get_state()
         vkd3d = self.switch_vkd3d.get_state()
         nvapi = self.switch_nvapi.get_state()
         fsr = self.switch_fsr.get_state()
         virt_desktop = self.switch_virt_desktop.get_state()
 
-        self.program["dxvk"] = dxvk
+        self.program["d8vk"] = d8vk
```
Comment (qarmin) [inline bottles/frontend/windows/launchoptions.py:200]: Is removing dxvk in this line intentional?

**Comment by SuperSamus** [comment]
D8VK is intending to merge with DXVK, https://github.com/AlpyneDreams/d8vk/issues/16.

**Comment by AlpyneDreams** [comment]
> D8VK is intending to merge with DXVK, [AlpyneDreams/d8vk#16](https://github.com/AlpyneDreams/d8vk/issues/16).

See doitsujin/dxvk#3411 now. Merge coming soon I expect.

**Comment by orowith2os** [comment]
Let's let DXVK handle this, so we don't do anything extra, and it all Just Works.

**Comment by orowith2os** [comment]
Closing as Not Planned, unless the merge for upstream gets denied at some point in the future, in which case, can reopen.

**Pull request closed without merging**